refinery/file_store/models.py

```diff
@@ -519,18 +527,18 @@ def get_datafile_url(self):
         :type self: A FileStoreItem instance
         :returns: A url for the given FileStoreItem or None
         """
-
         if self.is_local():
             return self.datafile.url
         else:
             # data file doesn't exist on disk
             if os.path.isabs(self.source):
                 # source is a file system path
-                logger.error("File not found at '%s'",
-                             self.datafile.name)
+                logger.error("File not found at '%s'", self.datafile.name)
                 return None
             else:
                 # source is a URL
+                if self.source.startswith('s3://'):
```

Comment:
It is not really an error but simply an indicator that a file is still being imported from S3. Also, I'm trying to reduce the amount of logging: there a lot of spurious messages that make logs hard to read.